### PR TITLE
Build script to build and validate package bloom

### PR DESCRIPTION
**Pull request by vikasgupta8**
Build script to build and validate package bloom.

### CODE REVIEW

File: b/bloom/bloom_ubi8.4.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+echo `pwd`
+
+# Ensure go.mod file exists
+go mod init $PACKAGE_PATH
+
+go get ./...
+
+go mod tidy
+
+# building with make
```
Comment (kandarpamalipeddi) [inline b/bloom/bloom_ubi8.4.sh:53]: As we are using `go test`, do we need `make` ?

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: b/bloom/bloom_ubi8.4.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : github.com/bits-and-blooms/bloom and github.com/willf/bloom are same
+# Version       : v2.0.3
+# Source repo   : https://github.com/bits-and-blooms/bloom
+# Tested on     : RHEL 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vikas Gupta <vikas.gupta8@ibm.com>
```
Comment (kandarpamalipeddi) [inline b/bloom/bloom_ubi8.4.sh:8]: Add `Language` field to script header.

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: b/bloom/bloom_ubi8.4.sh (hunk 1 above)
Comment (vikasgupta8) [reply b/bloom/bloom_ubi8.4.sh]: Removed make

File: b/bloom/bloom_ubi8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply b/bloom/bloom_ubi8.4.sh:8]: Added language.

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**